CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ## Unreleased(https://github.com/raster-foundry/raster-foundry/tree/develop)
 
 ### Added
+- Disable blog feed and intercom initialization using webpack override file [\#4162](https://github.com/raster-foundry/raster-foundry/pull/4162)
```

Comment:
"Added" in the sense of "made configurable where previously configuration was not available"

Reply:
No it's fine -- I just wanted to vocalize / confirm I was on the same page you were on since this new changelog world still feels a bit unstable. I forgot a question mark in the comment, sorry about that.